central/image/service/service_impl.go

```diff
@@ -414,31 +421,43 @@ func (s *serviceImpl) EnrichLocalImageInternal(ctx context.Context, request *v1.
 		if exists && !forceScanUpdate && !forceSigVerificationUpdate {
 			return internalScanRespFromImage(existingImg), nil
 		}
+
+		if exists {
```

Comment:
nit: maybe `existingImg, exists, err := s.datastore.GetImage(ctx, imgID)` should be something like:

```
var existingImg <whatever>
var err error
existingImg, imgExists, err = s.datastore.GetImage(ctx, imgID)
```

This way, we don't need to add a new var to mirror the one only exposed inside the `if`

Reply:
I like that, will be cleaner, will make the change